### PR TITLE
Updated for video.js 4.12 and contrib-ads 1.1

### PR DESCRIPTION
**Pull request by shawnbuso**
This behaves a little bit differently than the old video.js and contrib-ads versions on mobile, and I don't know if everyone will like it. To compensate for that I've tagged the current stuff 0.1.0 and will release the below merge as 0.2.0.

### CODE REVIEW

**Comment by shawnbuso** [comment]
Will resolve #27

File: examples/advanced/ads.js [hunk 1]
```diff
@@ -29,7 +30,13 @@ var Ads = function() {
   // Start ads when the video player is clicked, but only the first time it's
   // clicked.
   this.clickedOnce = false;
```
Comment (tensafefrogs) [inline examples/advanced/ads.js]: Why not remove the event listener once the click/tap happens?

You can use something like
```e.target.removeEventListener(e.type, arguments.callee);``` within the function to remove itself.

**Comment by tensafefrogs** [comment]
Comment is optional as it might add more complexity. LGTM.

**Comment by shawnbuso** [comment]
Added a new issue to clean up the samples to make it cleaner.

**Pull request merged**